### ISSUE TITLE
Fix Build Issues and Runtime Issues for Probe

### ISSUE THREAD

**Issue by jvettraino** [di2e/Argo#95]
There are currently build issues when running the build on windows
 * Unit Tests are failing
 * RPM is not supported on windows causing windows build to fail

Also the probe and responder configurations should have the defaults updated.

**Comment by jmsimpson68** [comment]
The "Windows" installer is the zip file that gets generated in the build

**Comment by jmsimpson68** [comment]
Do you want to put in a Windows installer into the build?  That would certainly take some resources. You might want to try a different branch to install that functionality. Right now I am OK with just except for the windows.

**Comment by jvettraino** [comment]
I updated the details in the ticket to clarify some things.  For the Unit Tests, I am going to take a look at what is failing and why, and if it is too tough to recode, can we comment out (and create ticket to fix)?

For the RPM (sorry about being vague), I just want the build to work on Windows, right now since it can't find the RPM tools it fails.  So just putting the RPM stuff in a profile so it will only run when building on linux.

**Comment by jmsimpson68** [comment]
Copy all that.  A Linux profile is a good idea. Plus get rid of the "release" profile that goes to the DI2E artifact repo.  

We should find some time to review the tests (which are non trivial) and figure out how to fix.  Yea we can comment then out but I'd like not too.

**Comment by jvettraino** [comment]
Yeah I am not thrilled about commented them out, but the main benefit to comment them out for now is so others can build this.  We don't want Px-ers (or others evaluating) trying to build this, having it fail, and then give back a negative report.
I fixed a couple of the tests that were easy enough to fix (I imagine they were only issues on Windows), then I have noted the tests that I commented out, and will create a ticket with those details.

You okay with that approach?  Agree about getting together to talk about how to fix the tests, lets plan for that next week.
